calico/felix/test/data/felix_section.cfg

```diff
@@ -9,3 +9,6 @@ MetadataPort = 246
 
 [log]
 InterfacePrefix = whatever
+
+[log]
+ReportingIntervalSecs = 5
```

Comment:
You've not got a test for setting the ttl in the config file. So at the moment this tests that ttl defaults correctly from the interval read from file. If testing that defaulting is not duplicated in another test, then make sure you still cover both after adding ttl to this test config file.